bench/locli/src/Cardano/Unlog/LogObject.hs

```diff
@@ -99,11 +99,16 @@ readLogObjectStream f okDErr anyOks =
                       (printf "Decode error while parsing %s:\n%s\non input:\n>>>  %s" f (Text.toString err) (Text.toString input))
                     _ -> True)
                . loBody)) .
-    filter ((\case
+    filter ((case loAnyLimit of
+              Nothing -> \case
+                LOAny{} -> False
+                _       -> True
+              Just constraint -> \case
                 LOAny laty obj ->
-                  if elem laty anyOks then True else
+                  if elem laty constraint
+                  then True else
```

Comment:
not just `elem laty anyOks || error (printf "...")` ? ;-)

Reply:
The idea is that having an explicit `True` sort of rhymes with other case branches.